js/__internal/grids/data_grid/grouping/module.ts

```diff
@@ -348,6 +348,22 @@ const onGroupingMenuItemClick = function (column, params) {
   }
 };
 
+const isGroupPanelVisible = (groupPanelOptions): boolean => {
+  const visible = groupPanelOptions?.visible;
+
+  if (visible === 'auto') {
+    return devices.current().deviceType === 'desktop';
+  }
+
+  return visible;
+};
```

Comment:
Minor comment.

Let's use the ternary operator here:
```suggestion
const isGroupPanelVisible = (groupPanelOptions): boolean => {
  const visible = groupPanelOptions?.visible;
  
  return visible === 'auto'
    ? devices.current().deviceType === 'desktop';
    : visible;
};
```